lib/galaxy/managers/jobs.py

```diff
@@ -1119,7 +1119,7 @@ def get_jobs_to_check_at_startup(session: galaxy_scoped_session, track_jobs_in_d
         # Filter out the jobs of inactive users.
         stmt = stmt.outerjoin(User).filter(or_((Job.user_id == null()), (User.active == true())))
 
-    return session.scalars(stmt)
+    return session.scalars(stmt).all()
```

Comment:
Maybe also add type annotation for the function return value?

Reply:
I really do not like these overly verbose annotations. What is the advantage over the inferred type ?
![Screenshot 2024-09-17 at 15 29 00](https://github.com/user-attachments/assets/db6229f4-fc47-498c-bb9b-ebec34a3cf70)
The advantage of not doing it is that any change in the sqlalchemy mechanics get propagates to us.

Reply:
It would help with code refactorings and when reviewing e.g. on GitHub, but it's fine if you prefer not to.